.github/workflows/build-rust.yml

```
@@ -60,3 +60,6 @@ jobs:
     - name: Build
       shell: bash
       run: cargo build --verbose --workspace --all-features --target ${{ join(matrix.targets, ' --target ') }}
+    - name: Post-Build Tests
+      shell: bash
+      run: cargo test
```

```suggestion
      run: cargo test --verbose --workspace --all-features --target ${{ join(matrix.targets, ' --target ') }}
```

@conradev is this what you're looking for?